src/compare-oracle-to-fallback/compare-oracle-to-fallback.js

```diff
@@ -0,0 +1,185 @@
+const ethers = require('ethers');
+const BigNumber = require('bignumber.js');
+const {
+  Finding, FindingSeverity, FindingType, getJsonRpcUrl,
+} = require('forta-agent');
+
+// load required shared types
+const {
+  LendingPoolAddressesProvider: lendingPoolAddressesProvider,
+  ProtocolDataProvider: protocolDataProviderAddress,
+} = require('../../contract-addresses.json');
+const { abi: protocolDataProviderAbi } = require('../../interfaces/AaveProtocolDataProvider.json');
+const { abi: priceOracleAbi } = require('../../interfaces/IPriceOracle.json');
+const { abi: priceOracleGetterAbi } = require('../../interfaces/IPriceOracleGetter.json');
+const { abi: lendingPoolAddressesProviderAbi } = require('../../interfaces/ILendingPoolAddressesProvider.json');
+const { compareOracleToFallback, aaveEverestId } = require('../../agent-config.json');
+
+// set up the an ethers provider
+// do not use ethers.providers.WebSocketProvider in production (there is no support)
+const jsonRpcProvider = new ethers.providers.JsonRpcProvider(getJsonRpcUrl());
+
+// NOTE: this value is imported from agent-config.json
+// percent error threshold over which we trigger alerts (2% error compared to the price oracle)
```

Comment:
"2% error" - maybe reword this to highlight that 2% is the current config file default, or don't mention the value at all?

Reply:
Removed reference to the value.